### PR TITLE
Edited Graph Explorer pages

### PR DESCRIPTION
**Pull request by Linda-Editor**
High-level copyedit.

### CODE REVIEW

**Comment by Linda-Editor** [comment]
@RabebOthmani 
Question: are we removing Objective-C from Graph Explorer? If so, I need to update the text to remove it.

**Comment by Linda-Editor** [comment]
Staged:
https://review.docs.microsoft.com/en-us/graph/graph-explorer/graph-explorer-overview?branch=linda-editor-explorer
https://review.docs.microsoft.com/en-us/graph/graph-explorer/graph-explorer-features?branch=linda-editor-explorer

File: concepts/graph-explorer/graph-explorer-features.md [hunk 1]
```diff
@@ -39,13 +38,13 @@ Graph Explorer includes an **Access token** tab that shows your access token whe
...
-For each REST API query you select or enter in Graph Explorer, you can find how to call that API in each of the four languages showcased under the **Code snippets** tab - C#, Java, JavaScript, and Objective-C. 
+For each REST API query that you select or enter in Graph Explorer, you can find how to call that API in each of the four languages showcased under the **Code snippets** tab: C#, Java, JavaScript, and Objective-C.
 
 ![Screenshot of Graph Explorer with the code snippets tab highlighted](./images/code-snippets.png)
 
-## UI component integration
+## Integrate UI components
 
-Graph Explorer includes several features to make implementing UI easier. Reuse these components in your apps too.
+Graph Explorer includes several features to make implementing the user interface (UI) easier. Reuse these components in your apps too.
```
Comment (Lauragra) [inline concepts/graph-explorer/graph-explorer-features.md:47]: For the developer audience, we don't need to spell out UI.

**Review by Lauragra — APPROVED** [review]

File: concepts/graph-explorer/graph-explorer-features.md (hunk 1 above)
Comment (Linda-Editor) [reply concepts/graph-explorer/graph-explorer-features.md]: Thanks, I changed it back and made a few more tweaks to both pages.

File: concepts/graph-explorer/graph-explorer-overview.md [hunk 2]
```diff
@@ -9,25 +9,25 @@ author: RabebOthmani
...
+- View and copy code snippets of each query that you run in C#, Java, and JavaScript.
 - Access Microsoft Graph Toolkit components and adaptive cards for some sample queries.
 - Share queries, including the request body and request headers.
 
 Graph Explorer handles the authentication process for you. Customize the experience by collapsing the sidebar or changing the theme.
 
 ## Get started
 
-Graph Explorer is a web application hosted on the [Microsoft Graph developer center](https://developer.microsoft.com/en-us/graph/graph-explorer). It's an open-source project,  and we welcome your contributions and feedback in the [GitHub repo](https://github.com/microsoftgraph/microsoft-graph-explorer-v4).
+Graph Explorer is a web application hosted on the [Microsoft Graph developer center](https://developer.microsoft.com/en-us/graph). It's an open-source project,  and we welcome your contributions and feedback in the [GitHub repo](https://github.com/microsoftgraph/microsoft-graph-explorer-v4).
```
[the remark this answers is not in the document]
Comment (Linda-Editor) [reply concepts/graph-explorer/graph-explorer-overview.md:30]: @RabebOthmani The original link went to Graph Explorer but the display text was for the dev center. We already have a link to Graph Explorer in the first paragraph, so, for SEO purposes, I updated the link with the dev center text to point to the dev center. If you prefer to have a link to Graph Explorer here instead (which would be the second occurrence on the page), I'll change the link to go with the Graph Explorer text and leave the dev center text without a link. Let me know what you'd like.

**Pull request merged**